packages/in_app_purchase/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.0.0
+
+* BREAKING CHANGE : Refactor to handle new `PurchaseStatus` named `canceled`. This means developers
```

Comment:
Nit: please update this to follow the new style guide: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style

So it should be:
```
* **BREAKING CHANGE**: Adds a new `PurchaseStatus` named [...]
```